src/WasBornAfterTest.java

```diff
@@ -0,0 +1,110 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import java.lang.String;
+
+class WasBornAfterTest {
+
+  @Test
+  void should_ReturnTrue_When_Person_WasBornAfter() {
+
+    //given
+    Person b = getPerson("Jane", "1998-04-09");
+    Person c = getPerson("Harry", "1980-09-25");
+    Person e = getPerson("Jack", "1996-08-20");
+
+
+    List<Person> personList = List.of(b, c, e);
+
+    LocalDate firstDate = LocalDate.parse("1980-01-01");
+
+    for (Person i : personList) {
```

Comment:
You can probably remove this loop as we verify the DOBs using the assertions instead